acceptance/bundle/deploy/fail-on-active-runs/script

```diff
@@ -0,0 +1,7 @@
+trace $CLI bundle deploy # We deploy the bundle first to create a job
+trace errcode $CLI bundle deploy --fail-on-active-runs # We deploy the bundle again to check that the deploy is failing if the job is running
+
+title "Expecting only 1 delete request to artifact_path/.internal folder"
```

Comment:
This request comes from the first deploy, correct?

Reply:
Yes, exactly, added to the title